<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_SoundcloudArtistSearch</class>
 <widget class="QDialog" name="GUI_SoundcloudArtistSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Soundcloud</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QListWidget" name="lwArtists"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labAlbumHeader">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labArtistCount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labTrackHeader">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labStatus">
       <property name="text">
        <string notr="true">Status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string notr="true">Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string notr="true">Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labPlaylistCount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="labTrackCount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labArtistHeader">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="lwPlaylists">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QListWidget" name="lwTracks">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search artist</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>lwArtists</tabstop>
  <tabstop>lwPlaylists</tabstop>
  <tabstop>lwTracks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
